code/ship/ship.cpp

```diff
@@ -9388,7 +9388,9 @@ int ship_create(matrix* orient, vec3d* pos, int ship_type, const char* ship_name
 
 	// Goober5000 - if no ship name specified, or if specified ship already exists,
 	// or if specified ship has exited, use a default name
-	if ((ship_name == NULL) || (ship_name_lookup(ship_name) >= 0) || (ship_find_exited_ship_by_name(ship_name) >= 0)) {
+	// Cyborg17 - The final check here was supposed to prevent duplicate names from being on the ship log and causing chaos,
+	// but it breaks multi,are just going to have a warning for testers to find, instead.
+	if ((ship_name == NULL) || (ship_name_lookup(ship_name) >= 0) /*|| (ship_find_exited_ship_by_name(ship_name) >= 0)*/) {
```

Comment:
a) NULL -> nullptr
b) mission log, not ship log
c) "are just going to have a warning"?  that doesn't make sense and should be reworded